index.js

```diff
@@ -292,6 +292,7 @@ class K8sExecutor extends Executor {
         this.privileged = hoek.reach(options, 'kubernetes.privileged', { default: false });
         this.secrets = hoek.reach(options, 'kubernetes.buildSecrets', { default: {} });
         this.secretsFile = hoek.reach(options, 'kubernetes.buildSecretsFile', { default: {} });
+        this.sdHabEnabled = options.sdHabEnabled || 'yes';
```

Comment:
`true` or `yes`?

Reply:
https://github.com/screwdriver-cd/launcher/pull/429#discussion_r747724559
Based on the feedback, I set it to `yes`